src/formElementPlugins/listBuilder/src/displayComponents/ListInlineLayout.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect, useState } from 'react'
+import { Button, Icon, Accordion } from 'semantic-ui-react'
+import { ApplicationDetails, ElementState, User } from '../../../../utils/types'
+import { ListLayoutProps } from '../types'
+import ApplicationViewWrapper from '../../../ApplicationViewWrapper'
+import SummaryViewWrapper from '../../../SummaryViewWrapper'
+import { buildElements, substituteValues } from '../helpers'
+import '../styles.less'
+
+const ListInlineLayout: React.FC<ListLayoutProps> = (props) => {
+  const { listItems, displayFormat } = props
+  return (
+    <>
+      {listItems.map((item, index) => (
+        <ItemAccordion
+          key={index} // Not ideal, but there isn't any other unique identifier
+          index={index}
+          item={item}
+          header={displayFormat.title}
+          {...props}
+        />
+      ))}
+    </>
+  )
+}
+export default ListInlineLayout
+
+// Inner component -- one for each Item in list
+const ItemAccordion: React.FC<any> = ({
```

Comment:
```suggestion
const ItemAccordion: React.FC<ListLayoutProps > = ({
```
?

Reply:
Not quite so straightforward, as it has some additional props above ListLayoutProps, but i've made it work: 21ecb3f7

Reply:
> Tested now after import of Lookup table ingredients.csv - suggestion add the file in the lookup table for snapshot dev so it's already there for future use.

Yeah, I'll do this when I update the dev snapshot overall with updated Feature Showcase and this listbuilder element, etc.